.github/workflows/smoke-test.yml

```diff
@@ -48,7 +45,9 @@ jobs:
 
       - name: Install
         run: |
-          sudo snap install --dangerous ./docker.snap
+          # dawidd6/action-download-artifact action creates a directory when the name is fetch as a RegExp
```

Comment:
```suggestion
          # dawidd6/action-download-artifact action creates a directory when the name is given as a regex
```

Reply:
To separate workflow logic from the test's set up, you could change the working directory from the step itself, moving the above comment there.

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory

Reply:
I'm using "RegExp" here to match the name of the option `name_is_regexp`, but I replace "fetch" -> "given"

The `working-directory` instruction doesn't accept wildcards, [I've just tested it here](https://github.com/locnnil/ci-exp/actions/runs/10549822156/job/29225312984#step:4:7).